src/OrbitQt/HistogramWidget.h

```diff
@@ -25,6 +26,8 @@ class HistogramWidget : public QWidget {
  public:
   using QWidget::QWidget;
 
+  void Initialize(OrbitApp* app) { app_ = app; }
```

Comment:
Do we allow `nullptr` being passed here? If not, consider adding an `ORBIT_CHECK(app != nullptr);`.